pgx-tests/src/tests/datetime_tests.rs

```diff
@@ -149,22 +148,9 @@ mod serialization_tests {
         let (h, ..) = time_with_timezone.to_utc().to_hms_micro();
         assert_eq!(10, h);
 
-        // b/c we always want our times output in UTC
-        assert_eq!(json!({"time W/ Zone test":"10:23:34-00"}), json);
+        // however Postgres wants to format it is fine by us
+        assert_eq!(json!({"time W/ Zone test":"12:23:34+02"}), json);
```

Comment:
Just pointing this out as a user-facing change.  I can't predict what this might mean but it's still ![technically correct](http://www.quickmeme.com/img/e2/e257f8ad4cb36f516fdf79be6cc3c49a7428a2e98c47e573e37c4f275eea0634.jpg).